test/controllers/api/v2/content_view_versions_controller_test.rb

```diff
@@ -158,6 +158,15 @@ def test_show_protected
       end
     end
 
+    def test_republish_repositories
+      version = @library_dev_staging_view.versions.first
+      @controller.expects(:async_task).with(::Actions::Katello::ContentViewVersion::RepublishRepositories, version).returns({})
+      post :republish_repositories, :id => version.id
```

Comment:
yeah it should, i think generally the rails tests don't actually care though.  I will update